js/Genoverse.js

```diff
@@ -155,7 +155,7 @@ const Genoverse = Base.extend({
       function init() {
         if (browser.loadedPlugins[plugin] !== true) {
           Genoverse.Plugins[plugin].call(browser, conf);
-          browser.container.addClass('gv-' + plugin.replace(/([A-Z])/g, '-$1').toLowerCase() + '-plugin');
+          browser.container.addClass('gvm-' + plugin.replace(/([A-Z])/g, '-$1').toLowerCase() + '-plugin');
```

Comment:
Can we leave a comment here for to say why this is a different class?

```suggestion
          // We use a different class here so this doesn't conflict with the old version of Genoverse running elsewhere
          // TODO: Remove this once we're only using one version, or find a different way to scope the styles eventually
          browser.container.addClass('gvm-' + plugin.replace(/([A-Z])/g, '-$1').toLowerCase() + '-plugin');
```

Reply:
Added your change. Good comment